xmlrpc/templates/xmlrpc-info.html

```diff
@@ -3,6 +3,9 @@
 {% block content %}
 <div class="section">
   <h2>XML-RPC Interface</h2>
-  <p>Interface with this page using an XML-RPC client, or use the <a href="{% url builder %}">interactive builder</a>. Visit the Seattle project wiki for the <a href="https://seattle.cs.washington.edu/wiki/CustomInstallerBuilderAPI">XML-RPC API</a>.</p>
+  <p>Interface with this page using an XML-RPC client, or use the 
+  <a href="{% url "builder" %}">interactive builder</a>.
```

Comment:
Seems to work. It finds the intended [url](https://github.com/aaaaalbert/custominstallerbuilder/blob/master/html/urls.py#L29) which leads to the intended [view](https://github.com/aaaaalbert/custominstallerbuilder/blob/master/html/views.py#L376) which renders the intended [html](https://github.com/aaaaalbert/custominstallerbuilder/blob/master/html/templates/builder.html).